android/app/src/main/java/com/expensify/chat/MainApplication.java

```diff
@@ -111,13 +110,10 @@ private static void initializeFlipper(
         aClass
             .getMethod("initializeFlipper", Context.class, ReactInstanceManager.class)
             .invoke(null, context, reactInstanceManager);
-      } catch (ClassNotFoundException e) {
-        e.printStackTrace();
-      } catch (NoSuchMethodException e) {
-        e.printStackTrace();
-      } catch (IllegalAccessException e) {
-        e.printStackTrace();
-      } catch (InvocationTargetException e) {
+      } catch (ClassNotFoundException
+              | NoSuchMethodException
+              | IllegalAccessException
+              | InvocationTargetException e) {
         e.printStackTrace();
```

Comment:
Lol. Maybe we could just catch every exception at this point? 

```
} catch (Exception e) {
```

Reply:
Done